common/lib/transport/connectionerror.js

```diff
@@ -38,4 +38,17 @@ var ConnectionError = {
 	})
 };
 
+ConnectionError.isRetriable = function(err) {
+	if (!err.statusCode || !err.code || err.statusCode >= 500) {
+		return true;
+	}
+	var retriable = false;
+	ConnectionError.keys().forEach(function(connErr) {
```

Comment:
> `ConnectionError.keys().forEach(function(connErr) {...`

That won't work for a couple reasons (plain objects don't have a `keys()` function, and anyway you want the values not the keys). You want something like `Utils.valuesArray(ConnectionError).forEach(function(connErr) { ...`

(I think IE9 probably has Array#forEach? double check, if it doesn't use `Utils.arrForEach`)